app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
Here I'd also name arguments appropriately, even though it's not as important as in the move-file task.